### PR TITLE
AB#49324-Create/Update Email Notification Templates on Application Startup

### PR DESCRIPTION
**Pull request by TCPrimedPaul**
We currently only have one template which is named: _Seafarers_Document_Submission_Email_.
The code being checked in here updates the template named _Seafarers_Document_Submission_Email_TESTING_ to not interfere with any ongoing changes to the template.
This code will need to be harmonized with Mansuer's MTOA work.

### CODE REVIEW

File: CDTS Blazor/Resources/EmailTemplates/SubmissionEmailTemplateSubjectTextEnglish.html [hunk 1]
```diff
@@ -0,0 +1 @@
+﻿Transport Canada has received your documents - Confirmation #{Confirmation_Number}
```
Comment (tc-tibo) [inline CDTS Blazor/Resources/EmailTemplates/SubmissionEmailTemplateSubjectTextEnglish.html:1]: Per the [EN Content guide](https://034gc.sharepoint.com/:w:/s/C4CSeafarerProject/ES5wHH7g8J5DqdtBhbYt3KMB003R_QYigjJTaeZeJKdoVA?e=c09IP4), we should make this:

`Transport Canada has received your documents / Transports Canada a reçu vos documents - Confirmation # {Confirmation_Number} `

image

File: CDTS Blazor/Resources/EmailTemplates/SubmissionEmailTemplateSubjectTextFrench.html [hunk 2]
```diff
@@ -0,0 +1 @@
+﻿Transports Canada a reçu vos documents - Confirmation #{Confirmation_Number}
```
Comment (tc-tibo) [inline CDTS Blazor/Resources/EmailTemplates/SubmissionEmailTemplateSubjectTextFrench.html:1]: Per [FR Content guide](https://034gc.sharepoint.com/:w:/s/C4CSeafarerProject/ETyMKmJe7WRFuQSegiqbNvkB_wNS_mz-nJlQLuRzWjG9rQ?e=1U7Z38), this should read:

`Transports Canada a reçu vos documents / Transport Canada has received your documents - Confirmation # {Confirmation_Number}`

image

File: CDTS Blazor/Resources/EmailTemplates/SubmissionEmailTemplateSubjectTextEnglish.html (hunk 1 above)
Comment (TCPrimedPaul) [reply CDTS Blazor/Resources/EmailTemplates/SubmissionEmailTemplateSubjectTextEnglish.html]: Change made.

**Comment by tc-tibo** [comment]
Everything looks really nice @TCPrimedPaul !

I like how you did this in the style of IaC. It also allows changes to the email templates to tracked in version control.

@AlexandreSanscartier mentioned he was going to review so I will leave approval for him after he's had a chance to look.

**Review by AlexandreSanscartier — APPROVED** [review]

**Pull request merged**